influx_interface.py

```diff
@@ -9,14 +11,24 @@
 # For development only
 import sys, traceback
 
+is_version2 = db_settings['version'] == 2
```

Comment:
This will raise a `KeyError` unless the replacement `db_settings` in lines 28-35 of config.py is uncommented.  Perhaps wrap in a try/except, or check for the existing of the `'version'` key first with ` if 'version' in db_settings.keys():`